src/test/scala/uk/gov/nationalarchives/notifications/GenericMessageIntegrationSpec.scala

```diff
@@ -0,0 +1,47 @@
+package uk.gov.nationalarchives.notifications
+
+import cats.implicits.catsSyntaxOptionId
+import org.scalatest.prop.TableFor7
+
+class GenericMessageIntegrationSpec extends LambdaIntegrationSpec {
+
+
+  override def events: TableFor7[String, String, Option[String], Option[String], Option[SqsExpectedMessageDetails], () => Unit, String] = {
+    val oneMessage: String =
+      """ "{\"messages\":[{\"message\":\"A test message\"}]}" """
+    val successfulMessageText = "A test message"
+    val twoMessages: String =
+      """ "{\"messages\":[{\"message\":\"A test message\"}, {\"message\":\"A second test message\"}]}" """
+    val failedMessageText = "A test message\\nA second test message"
+
+    Table(
+      ("description", "input", "emailBody", "slackBody", "sqsMessage", "stubContext", "slackUrl"),
+      ("a single message",
+        secretRotationEventInput(oneMessage), None, slackMsg(successfulMessageText).some, None, () => (), "/webhook"),
+      ("multiple messages",
```

Comment:
Should the description be "failed message", or something like that, as this is returning the `failedMessageText`

Reply:
It's the `failedMessageText` that doesn't make sense so I've changed it to `twoMessagesText`